### DISCUSSION TITLE
How to test if a media_player is compatible!

### DISCUSSION THREAD

**Discussion by erkr** [music-assistant/support#622] [Show and tell]
Music Assistant only accepts media players that support playing URL's. 

- First situation: The player is not in the list during configuration (so most likely it is incompatible) But if you have doubts, the most simple test is to check if the player works with mp3 files or radio stations via the standard media browser. If that works, and it is not listed by MA, you can open an issue that it should be listed.

- Second situation: The player is listed, but doesn't work. In that case (after you checked if  playback works in the standard media browser), there is an expert setting that can probably fix the playback: 
Open the player settings in MA for the one that doesn't work. For DLNA players try queue flow mode. For Airplay devices try using encryption.

- Third situation: none of the players in MA work. In that case there is probably a networking/HA configuration issue. That can be caused by running a custom setup and/or firewalls that block the streams. 
But if you are running a plain HAOS setup, then check your internal URL setting. That URL is used by MA to resolve your internal IP address. MA by default uses TCP port 8095. That port should not be blocked either.

**Comment by OzGav** [comment]
Relates to v1
